.travis.yml

```diff
@@ -1,12 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-    - "0.10"
-    - "0.11"
-    - "0.12"
-    - "iojs"
-matrix:
-    allow_failures:
-        - node_js: "0.11"
-        - node_js: "0.12"
-        - node_js: "iojs"
+    - "4"
```

Comment:
can we still test against 0.10, 0.12? we probably shouldn't allow failures for them anymore